src/nunavut/jinja/__init__.py

```diff
@@ -779,9 +779,11 @@ def __init__(self, namespace: nunavut.Namespace, **kwargs: typing.Any):
         target_language = self.language_context.get_target_language()
 
         self._sub_folders = None  # type: typing.Optional[pathlib.Path]
-        self._support_enabled = False  # If not enabled then we remove any support files found
+        self._serialization_support_enabled = (
+            False  # If not enabled then we remove any serialization support files found
+        )
         if target_language is not None:
-            self._support_enabled = not target_language.omit_serialization_support
+            self._serialization_support_enabled = not target_language.omit_serialization_support
```

Comment:
previously we only has support files that were for serialization so we'd omit generating them if serialization was turned off. This change allows us to keep disabling serialization support generation but always generate support types used by generated objects.